AllReadyApp/NotificationsProcessor/Functions.cs

```diff
@@ -47,6 +47,7 @@ public static void ProcessEmailQueueMessage([QueueTrigger("email-pending-deliver
             myMessage.AddTo(emailMessage.Recipient);
             myMessage.From = new MailAddress(from, "AllReady");
             myMessage.Subject = emailMessage.Subject;
+            myMessage.Html = emailMessage.HtmlMessage;
```

Comment:
@tonysurma This is what is passed to SendGrid. We **really** want to think about refining these messages.